app/views/prison/dashboards/show.html.erb

```diff
@@ -1,32 +1,32 @@
 <%= render 'navigation', estate: @estate, visits: @requested_visits %>
 
-<h2 class="heading-large">Latest visit requests</h2>
+<h2 class="heading-large"><%= t('.title') %></h2>
 
 <% if @requested_visits.any? %>
 
   <table>
-    <caption class="visuallyhidden">Visit inbox</caption>
+    <caption class="visuallyhidden"><%= t('.title') %></caption>
     <thead>
       <tr>
-        <th>Prisoner No.</th>
-        <th>Prisoner Name</th>
-        <th>Visit Date</th>
+        <th>Prisoner name</th>
+        <th>Prisoner no.</th>
+        <th>Visit date</th>
         <th>Time slot</th>
-        <th>Lead Visitor</th>
+        <th>Lead visitor</th>
```

Comment:
I might have not been clear @chrishanes when discussing locales, @mloughran is right and everything should be on locale files.